namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-person-record-dev/resources/irsa.tf

```diff
@@ -2,8 +2,9 @@ locals {
   # *** Placeholder for incoming SQS queues ***
   sqs_queues = {
   }
-  # *** Placeholder for incoming SNS topics ***
   sns_topics = {
+    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-dev"
+    "cloud-platform-probation-in-court-team-5b4824dca700d8b3ec75f25d24adfbb9"= "court-probation-dev"
```

Comment:
I am not clear on what the values after the = are supposed to be. Hopefully they are the namespace where the topic is defined :-)